metamorph/store/dynamodb/dynamodb.go

```diff
@@ -19,62 +19,70 @@ import (
 	"github.com/ordishs/gocore"
 )
 
+const (
+	ISO8601      = "2006-01-02T15:04:05.999Z"
```

Comment:
Why do we need to keep this custom format everywhere?
What is wrong with `RFC3339Nano = "2006-01-02T15:04:05.999999999Z07:00"`

https://pkg.go.dev/time#:~:text=The%20RFC3339Nano%20format%20removes%20trailing,creating%20a%20custom%20layout%20string.